zuul/connection/github.py

```diff
@@ -220,6 +220,13 @@ def _pull_request_to_event(self, pr_body):
 
         event.title = pr_body.get('title')
 
+        # get the statuses
+        project_name = head.get('repo').get('full_name')
+        head_owner, head_project = project_name.split('/')
+        event.statuses = self.connection.getCommitStatuses(head_owner,
```

Comment:
So attempting to query statuses on the PR (org/repo + sha) doesnt return anything, it needs to be queries against the upstream (org/repo + sha)